compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -107,7 +107,11 @@ fn reachable_non_generics_provider(tcx: TyCtxt<'_>, _: LocalCrate) -> DefIdMap<S
         .map(|def_id| {
             // We won't link right if this symbol is stripped during LTO.
             let name = tcx.symbol_name(Instance::mono(tcx, def_id.to_def_id())).name;
-            let used = name == "rust_eh_personality";
+            // We have to preserve the symbols of the built-in functions during LTO.
+            let is_builtin_fn = is_compiler_builtins
+                && symbol_export_level(tcx, def_id.to_def_id())
+                    .is_below_threshold(SymbolExportLevel::Rust);
```

Comment:
I think you mean `SymbolExportLevel::C`. The current code will make all functions in compiler-builtins `#[used]` rather than just the `#[no_mangle]` functions.

Reply:
Thanks! Yes!